pyproject.toml

```diff
@@ -91,7 +91,7 @@ description = "Env for running development commands for testing"
 features = ["test"]
 
 [tool.hatch.envs.test.scripts]
-unit = "- python -m pytest {args:tests/unit}"
```

Comment:
This prevents other commands from not running but it makes it look like a success so CI was not failing when it should